tests/Unit/UtilizationMetricsApisTest.php

```diff
@@ -0,0 +1,436 @@
+    $softDeleteRecruitmentCrmUser->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['recruitment_crm_users'])->toBe($randomRecords);
+});
+
+it('checks the API returns Student Records', function () {
+    $randomRecords = random_int(1, 10);
+
+    $totalRecords = $randomRecords + (Student::count());
+
+    Student::factory()->count($randomRecords)->create();
+
+    $softDeleteStudent = Student::factory()->create();
+    $softDeleteStudent->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['student_records'])->toBe($totalRecords);
+});
+
+it('checks the API returns Prospects Records', function () {
+    $randomRecords = random_int(1, 10);
+
+    Prospect::factory()->count($randomRecords)->create();
+
+    $softDeleteProspect = Prospect::factory()->create();
+    $softDeleteProspect->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['prospect_records'])->toBe($randomRecords);
+});
+
+it('checks the API returns Campaigns', function () {
+    $randomRecords = random_int(1, 10);
+
+    Campaign::factory()->count($randomRecords)->create();
+
+    $softDeleteCampaign = Campaign::factory()->create();
+    $softDeleteCampaign->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['campaigns'])->toBe($randomRecords);
+});
+
+it('checks the API returns Journey Steps Executed', function () {
+    $randomRecords = random_int(1, 10);
+
+    CampaignAction::factory()->count($randomRecords)->create();
+
+    $softDeleteCampaignAction = CampaignAction::factory()->successfulExecution()->create();
+    $softDeleteCampaignAction->delete();
+
+    CampaignAction::factory()->count($randomRecords)->successfulExecution()->create();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['journey_steps_executed'])->toBe($randomRecords);
+});
+
+it('checks the API returns Tasks', function () {
+    $randomRecords = random_int(1, 10);
+
+    Task::factory()->count($randomRecords)->create();
+
+    $softDeleteTask = Task::factory()->create();
+    $softDeleteTask->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['tasks'])->toBe($randomRecords);
+});
+
+it('checks the API returns Alerts', function () {
+    $randomRecords = random_int(1, 10);
+
+    Alert::factory()->count($randomRecords)->create();
+
+    $softDeleteAlert = Alert::factory()->create();
+    $softDeleteAlert->delete();
+
+    $response = get('/api/utilization-metrics');
```

Comment:
We should be using the route name here, not calling the path directly.

Reply:
Done